src/helper/apiHelper.js

```diff
@@ -55,7 +55,7 @@ module.exports = class ApiHelper {
 			// we got a non-200 response which can be responded but will still be thrown
 			throw this.transformResponse(error.response);
 		}
-		if (error.request) {
+		if (!error.request) {
```

Comment:
To me it looks like it was OK before you change it.
If we have response, we got a 4xx or 5xx response.
If we have no response, but a request, we have a timeout or some sort of network error.
If we have neither request nor response something else went wrong.

I found the following code snippet [here](https://www.intricatecloud.io/2020/03/how-to-handle-api-errors-in-your-web-app-using-axios/)
 ```
if (error.response) {
      // client received an error response (5xx, 4xx)
    } else if (error.request) {
      // client never received a response, or request never left
    } else {
      // anything else
    }
```
(since we throw in each case the else can be omitted)

Reply:
sure just reminded on that when we were discussing the nuxt part about it and missed that this is request, not response... will close the pr. @pkleybolte